app/experimenter/static/js/utils/api.js

```diff
@@ -1,6 +1,6 @@
 import { request } from "experimenter/utils/request.js";
 
-const API_ROOT = "/api/v1/";
+const API_ROOT = "/api/v2/";
```

Comment:
Oh cool!!! You can just change this here and all the frontend is updated.  I LOVE IT 💯 💯 💯